packages/content/src/content.ts

```diff
@@ -62,6 +66,97 @@ function getContentData(
   return getItemData(content.id, requestOptions);
 }
 
+interface IMetadataPaths {
+  updateFrequency: string;
+  reviseDate: string;
+  pubDate: string;
+  createDate: string;
+}
+
+function getMetadataPath(identifier: keyof IMetadataPaths) {
+  // NOTE: i have verified that this will work regardless of the "Metadata Style" set on the org
+  const metadataPaths: IMetadataPaths = {
+    updateFrequency:
+      "metadata.metadata.dataIdInfo.resMaint.maintFreq.MaintFreqCd.@_value",
+    reviseDate: "metadata.metadata.dataIdInfo.idCitation.date.reviseDate",
+    pubDate: "metadata.metadata.dataIdInfo.idCitation.date.pubDate",
+    createDate: "metadata.metadata.dataIdInfo.idCitation.date.createDate"
+  };
+
+  return metadataPaths[identifier];
+}
+
+function getValueFromMetadata(
+  content: IHubContent,
+  identifier: keyof IMetadataPaths
+) {
+  const path = getMetadataPath(identifier);
+  return path && getProp(content, path);
+}
+
+/**
+ * Enriches the content with additional date-related information
+ * Note that this is exported to facilitate testing but it should be considered private
+ *
+ * @private
+ * @param {IHubContent} content - the IHubContent object
+ * @returns {IHubContent}
+ */
+export function _enrichDates(content: IHubContent): IHubContent {
+  const newContent = cloneObject(content);
+
+  // updateFrequency:
+  const updatedFrequencyValue = getValueFromMetadata(
+    newContent,
+    "updateFrequency"
+  );
+  if (updatedFrequencyValue) {
+    const updateFrequencyMap = {
```

Comment:
Consider making the value an exported type that just is a list of the possible strings... that will allow consumers to know all the possible values and setup i18n for them all